### PR TITLE
Josh

### PR DESCRIPTION
**Pull request by thonnissen**
formatted design spec and deleted unused files

### CODE REVIEW

File: DesignSpecification.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Create an app that allows people to request deliveries and errands to be done for them.
+##### Design Specifications
+
+###### Mandatory Requirements:
+  * *A user should be able to create, read, update, display and delete a request.* 
+    * Once a user enters the request page they should be required to submit their contact email and phone number. 
+    
+    **PLACE WIREFRAME OF SCREEN FOR CONTACT INFORMATION TO BE LOGIN LATER HERE**
+    * They should then be directed to another page to enter a mandatory pickup address along with optional field for special pickup instructions. They should also have a mandadtory field for delivery address and optional field for special delivery instructions. There should also be a link to upload images of reciets or order confirmations for the driver.
```
Comment (MLGainz) [inline DesignSpecification.md]:
```suggestion
    * They should then be directed to another page to enter a mandatory pickup address along with optional field for special pickup instructions. They should also have a mandatory field for delivery address and optional field for special delivery instructions. There should also be a link to upload images of receipts of order confirmations for the driver.
```

File: DesignSpecification.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    
+    **PLACE WIREFRAME OF SCREEN FOR DELIVERY REQUEST INFORMATION HERE**
+    * For each request a user shall be able to optionally select a desired delivery time.
+    * Once a request is submitted the user shall see a confirmation page where they can confirm information and change errors.
+    
+    **PLACE WIREFRAME OF SCREEN FOR REQUEST SUBMISSION PAGE HERE**
+  * *A driver has the option to accept any request in the display menu.*
+    * There shall be a page the driver can access and veiw a lost of open request within their area of delivery and accept a delivery.
```
Comment (MLGainz) [inline DesignSpecification.md]:
```suggestion
    * There shall be a page the driver can access and view a list of open requests within their area of delivery and accept a delivery. 
```

File: DesignSpecification.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    **PLACE WIREFRAME OF SCREEN FOR DELIVERY REQUEST INFORMATION HERE**
+    * For each request a user shall be able to optionally select a desired delivery time.
+    * Once a request is submitted the user shall see a confirmation page where they can confirm information and change errors.
+    
+    **PLACE WIREFRAME OF SCREEN FOR REQUEST SUBMISSION PAGE HERE**
+  * *A driver has the option to accept any request in the display menu.*
+    * There shall be a page the driver can access and veiw a lost of open request within their area of delivery and accept a delivery. 
+    
+    **PLACE WIREFRAME OF SCREEN FOR LIST OF REQUESTS HERE**
+    * Once accepted there should be a notification sent to customer of the driver that will be working their request. The driver shall have an option to contact the client if desired.
```
Comment (MLGainz) [inline DesignSpecification.md]:
```suggestion
    * Once accepted, there should be a notification sent to the customer and the driver that will be working their request. The driver shall have an option to contact the client if desired.
```

File: DesignSpecification.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  * *After a delivery has been done. A delivery confirmation screen will show up for both the user and the driver.*
+  
+  **PLACE WIREFRAME OF SCREEN FOR CONFIRMATION PAGE HERE**
+  
+###### Desirable requirements:
+  * The user should be able to see real time location of the driver
+  * The driver should be able to access a gps through the app to locate his delivery and pickup addresses (Google Maps        integration)
+  * The user and the driver should have the ability to rate each other after a delivery request has been done and the confirmation page popped up.
+  * Every user and driver should be able to create and account with their email and password.
```
Comment (MLGainz) [inline DesignSpecification.md]:
```suggestion
  * Every user and driver should be able to create an account with their email and password.
```

File: DesignSpecification.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  * Drivers should recieve notifications notifying them about any delivery requests within their radius.
+  * After a Delivery request has been done. An Email confirmation should be sent to their email showing a sumary of the
```
Comment (MLGainz) [inline DesignSpecification.md]:
```suggestion
  * After a Delivery request has been done. An Email confirmation should be sent to their email showing a summary of the
```

**Comment by sjiang1** [comment]
Overall, very nice work!

**Pull request merged**